sofascoreAcademy/sofascoreAcademy/ViewModel/MatchViewModel.swift

```diff
@@ -26,22 +25,22 @@ struct MatchViewModel {
     
     var timeStatusText: String {
         switch event.status {
-        case .notStarted:
+        case .NOT_STARTED:
```

Comment:
Treba vratiti kako je bilo, ovakav naziv nije praksa u swiftu. Preparsiraj to sto dobijes od backenda u vrijednosti koje si imala.